settings/settings.go

```diff
@@ -59,13 +59,13 @@ func Load() (*Settings, error) {
 
 	c, err := deis.New(sF.VerifySSL, sF.Controller, sF.Token)
 
-	// Set a custom user agent
-	c.UserAgent = UserAgent
-
 	if err != nil {
 		return nil, err
 	}
 
+	// Set a custom user agent
+	c.UserAgent = UserAgent
```

Comment:
I'm not going to merge the first commit in this PR. I'll merge #123 and then rebase to clean up history.